index.js

```diff
@@ -20,7 +20,11 @@ const {
   getOptions,
   getSurveyMetadata,
   listSurveyVersion,
-  getSurveyVersion
+  getSurveyVersion,
+  listLibraryMessages,
+  getLibraryMessage,
```

Comment:
Maybe it'll be better for `index.js` to be gitignored, since it's just used for testing.